Gems/Atom/RHI/Code/Include/Atom/RHI/Factory.h

```diff
@@ -11,11 +11,15 @@
 #include <Atom/RHI/PhysicalDevice.h>
 #include <AzCore/Debug/Budget.h>
 #include <AzCore/Name/Name.h>
+#include <AzCore/Console/IConsole.h>
 
 #if defined(USE_RENDERDOC)
 #include <renderdoc_app.h>
 #endif
 
+AZ_CVAR(bool, r_gpuMarkersMergeGroups, false, nullptr, AZ::ConsoleFunctorFlags::Null, "Enable merging of gpu markers in order to track payload (i.e all the scopes) per command list.");
+
```

Comment:
@kberg-amzn Do CVAR's in headers cause issues across modules?

Reply:
Nope, you should put the cvar itself in a .cpp and extern it in a header.. There's a helper macro for externing a cvar. Technically you *can* put the cvar in the header, but it will generate many copies of that cvar for every .cpp you include that header in and then combine them at runtime, which isn't the greatest efficiency-wise.

Reply:
I made this change. Thanks for the suggestion.